ga4gh/frontend.py

```diff
@@ -111,11 +149,16 @@ def handleException(exception):
     return response
 
 
-def handleFlaskRequest(flaskRequest, endpoint):
+def handleFlaskPostRequest(version, flaskRequest, endpoint):
+    if Version.parseString(version) != Version.parseString(protocol.version):
```

Comment:
Oh yeah - that 'v' is very nasty. OK, let's keep the class and just store a tuple internally then.